### ISSUE TITLE
How to force to remember window position?

### ISSUE THREAD

**Issue by silentmantra** [mintty/wsltty#331]
First thank you for the best wsl terminal (the miserable windows terminal goes to hell).
I have 2 questions:
1. how to get wsltty to remember window position? after launch i always get small window at the top left which is annoying. and probably this should be done per a config file maybe or an argument list? so i would like to have several shorcuts in the taskbar each would remember its own window position.
2. is it possible to define command which will be executed in wsl on the start? in the argument list or config.
Thanks!

**Comment by silentmantra** [comment]
seems that's in the mintty's config settings.
will try

**Comment by Biswa96** [comment]
I think you have to add `--position` command line option in the mintty/wsltty shortcut. The details can be found here https://mintty.github.io/mintty.1.html

**Comment by silentmantra** [comment]
> I think you have to add `--position` command line option in the mintty/wsltty shortcut. The details can be found here https://mintty.github.io/mintty.1.html

yes, works fine, cannot figure out how to execute bash command on start though... `--exec` doesnt work for some reason, says "wrong directory"

**Comment by Biswa96** [comment]
> cannot figure out how to execute bash command on start though...

In the wsltty shortcuts properties, try to replace the `-` with the command. @mintty may provide proper solution.

**Comment by mintty** [comment]
Replace the `--WSL=...` parameter with just `--wsl`.
Replace the final `-` with the wslbridge2 command, including the distro, and add the desired command, e.g.
`wslbridge2 -d Debian mc`
